torch/onnx/symbolic_opset10.py

```diff
@@ -9,6 +9,7 @@
 import torch.onnx.symbolic_helper as sym_help
 from torch.onnx.symbolic_helper import parse_args, _unimplemented
 import torch.onnx.symbolic_opset9
+from torch.onnx.symbolic_opset9 import _compatible_float_cast
```

Comment:
```suggestion
from torch.onnx.symbolic_opset9 import op_with_optional_float_cast
```
Methods names are clearer when they start with either verb or a noun (when something is being created created).